### PR TITLE
Removed bootstrap added govuk framework

### PR DESCRIPTION
**Pull request by dp6ai**

### CODE REVIEW

**Comment by dp6ai** [comment]
I'm going to go through and tidy this up some more. i wanted to get it rebased and on the test heroku site so people could start looking at it.

File: app/assets/javascripts/dsds.js.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    var own_setup = function () {
+        $('#defence_request_solicitor_firm').removeAttr("disabled");
+        $('#defence_request_solicitor_name').removeAttr("disabled");
+        $('#defence_request_scheme').val("No Scheme");
+        $('#defence_request_scheme').attr("disabled", "disabled");
+    }
+
+    var duty_setup = function () {
+        $('#defence_request_solicitor_firm').attr("disabled", "disabled");
```
Comment (pixeltrix) [inline app/assets/javascripts/dsds.js.erb]: You could cache these repeated jQuery lookups by saving it to a local variable but it's probably not worth it for alpha

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,12 +1,8 @@
 module ApplicationHelper
-  def flash_messages(opts = {})
-    flash.to_h.slice('notice', 'alert').each do |msg_type, message|
-      concat(content_tag(:div, message, class: "alert flash #{bootstrap_class_for(msg_type)} fade in") do
-        concat content_tag(:button, 'x', class: 'close', data: {dismiss: 'alert'})
-        concat safe_join(Array(message), tag(:br))
-      end)
-    end
-    nil
+  include ActiveModel
```
Comment (pixeltrix) [inline app/helpers/application_helper.rb]: Are you sure you want to do this? Including the whole of Active Model in the application helper?

File: config/application.rb [hunk 3]
```diff
@@ -28,7 +28,23 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
+    config.relative_url_root = ENV['RAILS_RELATIVE_URL_ROOT'] || ''
```
Comment (pixeltrix) [inline config/application.rb]: Why are you adding this? Are you planning to make it mountable under a subdirectory?

File: config/application.rb [hunk 4]
```diff
@@ -28,7 +28,23 @@ class Application < Rails::Application
...
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance|
+      html_tag
```
Comment (pixeltrix) [inline config/application.rb]: Why are you overriding the error proc?

File: config/initializers/simple_form.rb [hunk 5]
```diff
@@ -0,0 +1,166 @@
+# Use this setup block to configure all options available in SimpleForm.
+SimpleForm.setup do |config|
```
Comment (pixeltrix) [inline config/initializers/simple_form.rb]: Are there custom wrappers for the gov.uk toolkit that allow us to write less boilerplate - like how you can generate ones for Bootstrap.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (dp6ai) [inline app/helpers/application_helper.rb]: sorry. that should have been removed...

File: app/assets/javascripts/dsds.js.erb (hunk 1 above)
Comment (dp6ai) [inline app/assets/javascripts/dsds.js.erb]: :+1:

File: config/initializers/simple_form.rb (hunk 5 above)
Comment (dp6ai) [inline config/initializers/simple_form.rb]: config.use_govuk_elements_form_builder = true 
turns on the gds form builder. ill add it in the next story which involves styling the form further

File: config/application.rb (hunk 3 above)
Comment (dp6ai) [inline config/application.rb]: required for 
config.feedback_url = config.relative_url_root + '/feedback/new'
its all GDS standards

File: config/application.rb (hunk 4 above)
Comment (dp6ai) [inline config/application.rb]: taken from the internal app we are using to pull a lot of the styling form 
    ```
# disable default <div class="field_with_errors"> wrapping idiocy
    config.action_view.field_error_proc = Proc.new { |html_tag, instance|
      html_tag
    }
```

**Pull request merged**